src/api/providers/cline.ts

```diff
@@ -163,7 +163,8 @@ export class ClineHandler implements ApiHandler {
 			}
 		} catch (error) {
 			console.error("Cline API Error:", error)
-			throw error
+			// Preserve response headers
+			throw new Error(error.message, { cause: error })
```

Comment:
```suggestion
			throw error
```

Why would throwing the original error as-if would not preserve the response headers if it was actually included? We do get the request-id for some cases so I'm worried this change would cause regression. Can you confirm if this is working for you?

Reply:
This appears to be working fine for me -- I forced the API to return a 500 and see this in the extension
<img width="449" height="151" alt="image" src="https://github.com/user-attachments/assets/d79d12d0-3b36-4eb4-837c-1113c92031fc" />